src/RedirectBundle/Form/Type/RedirectType.php

```diff
@@ -32,6 +32,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('permanent', CheckboxType::class, [
                 'attr' => [
                     'align_with_widget' => true,
+                    'required' => false,
```

Comment:
Shouldn't the `required` option be defined on the toplevel of the options array? So above the `attr`, instead of inside the `attr`.

As far as I know the `attr` options are only values that will be added as html attributes to the input, but won't effect any backend validation